dashboard/app/views/home/_homepage.html.haml

```diff
@@ -42,3 +42,19 @@
       = link_to pd_workshop_admins_path do
         %h3
           Directory for Workshop Admins
+    %h1
+      Facilitator Landing Page
```

Comment:
Could you display this section only if the user has facilitated a course? For non-facilitator users, they will see this header without anything under it, which could be a bit confusing.

Reply:
Good point. I added a check to see if the list of courses facilitated is empty